Models/articles-model.js

```diff
@@ -31,7 +31,21 @@ function fetchArticles(){
     })
 }
 
+function fetchCommentsByID(article_id){
+    let query = "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;"
+
+    return db.query(query, [article_id]).then(({rows}) => {
+        if (rows[0] === undefined) {
+            return Promise.reject({
```

Comment:
You shouldn't need this logic here, if there are no comments then returning `rows` will return `[]` which is all we need